packages/babel-preset-env/src/available-plugins.js

```diff
@@ -49,7 +49,20 @@ import transformTemplateLiterals from "@babel/plugin-transform-template-literals
 import transformTypeofSymbol from "@babel/plugin-transform-typeof-symbol";
 import transformUnicodeRegex from "@babel/plugin-transform-unicode-regex";
 
+import bugfixAsyncArrowsInClass from "@babel/preset-modules/lib/plugins/transform-async-arrows-in-class";
```

Comment:
nit: We should export theses plugins from `@babel/preset-modules` instead of relying on internal file structures.

Reply:
I would like to just move the plugin files inside `@babel/preset-env` in the future, if @developit is ok with that.